tests/testthat/test-utils_insert_db.R

```diff
@@ -19,7 +19,7 @@ test_that("utils_insert_db functions other than dbUpdate", {
   # load pkg info for stringr into the database
   pkg_name <- "stringr"
   
-  insert_pkg_info_to_db(pkg_name, app_db_loc)
+  insert_pkg_info_to_db(pkg_name,, app_db_loc)
```

Comment:
It's not a typo because when running the function in testing mode it defaults to grabbing description information from CRAN. So I am intentionally not passing any values for version, but I can see when it looks wrong or confusing. I can pass `app_db_loc` as a named argument to increase readability.